packages/loopring_v3/circuit/Gadgets/SignatureGadgets.h

```diff
@@ -78,6 +79,10 @@ class RequireValidPublicKey : public GadgetT {
   }
 };
 
+// QUESTION(daniel): seems no output, so the compression is just check if we can
+// drop publickKeyX or publicKeyY entirely?
```

Comment:
That's right.

Reply:
Currently I'm still thinking no, it's still easier and cheaper to store publickKeyX/publickKeyY directly in the Merkle tree. The compressed form is then only used to put into the DA (so we only use 32 bytes, not 64 bytes) so we can reconstruct the publickKeyX/publickKeyY from it to make sure the fulll Merkle tree can be reconstructed. And even the compressed form is bigger than a single field element, so in any case we need to take up 2 fields elements in the Account leaf as we do now, so we may as well store the uncompressed form.

Reply:
Will this be in another PR or it is already the case (that we store both X and Y).

Reply:
Missed this one for some reason (didn't get an email notification that I can see). We already store both X and Y separately in the account leaf, we always have so no further changes.